Src/CSharpier.Rider/src/main/java/com/intellij/csharpier/CSharpierProcessPipeMultipleFiles.java

```diff
@@ -21,8 +21,9 @@
     public CSharpierProcessPipeMultipleFiles(String csharpierPath, boolean useUtf8) {
         this.csharpierPath = csharpierPath;
         try {
-            this.process = new ProcessBuilder("dotnet", csharpierPath, "--pipe-multiple-files")
-                    .start();
+            ProcessBuilder processBuilder = new ProcessBuilder(csharpierPath, "--pipe-multiple-files");
```

Comment:
I think you can use `var` as of Java 10.

Reply:
You sir, are a life saver! I will convert everything to vars in another PR. Weird that intellij wasn't suggesting this to me.